sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -1879,5 +1880,27 @@ class JDBCSuite extends QueryTest
     val fields = schema.fields
     assert(fields.length === 1)
     assert(fields(0).dataType === StringType)
-   }
+  }
+
+  test("SPARK-39339: TimestampNTZType support") {
+    val tableName = "timestamp_ntz_table"
```

Comment:
Could you add tests write/read timestamp_ntz with different time zone ? I doubt the result is not correct.

Reply:
Timestamp NTZ type is independent of the time zone. Sure, I can add a test to confirm.